webpack.config.js

```diff
@@ -1,12 +1,15 @@
 const path = require('path')
 const HtmlWebPackPlugin = require('html-webpack-plugin')
+const history = require('connect-history-api-fallback')
+const convert = require('koa-connect')
```

Comment:
Hva gjør koa-connect?

Reply:
Godt spørsmål, men det funker ikke uten.. 
Bruker dem her: 
`line 53:` `add: (app) => {
            app.use(convert(history({})))
        },
`

Prøvde uten både `convert` og `history`, men krevde at jeg brukte dem for å få routingen til å funke. Man kan benytte history til mer, men det hadde ikke vi behov for. Kan leses mer her: https://github.com/bripkens/connect-history-api-fallback#options